### PR TITLE
Integration test improvements

### PR DESCRIPTION
**Pull request by ludeeus**
## Proposed change

- Only run tests against "dev" for the scheduled action run
- On failure print the home assistant log file
- On failure for the scheduled run, post a message to discord.

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (which adds functionality to an existing integration)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [x] Code quality improvements to existing code or addition of tests

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] Local tests pass. **Your PR cannot be merged unless tests pass**
- [ ] There is no commented out code in this PR.
- [ ] I have followed the [development checklist][dev-checklist]
- [ ] The code compiles without warnings (code quality chek)
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/net-daemon/docs

### CODE REVIEW

**Review by helto4real — APPROVED** [review]
This is useful to keep track of errors more easy

**Comment by ludeeus** [comment]
@codecov-commenter, you drunk...

File: .github/workflows/ci_integration_tests.yaml [hunk 1]
```diff
@@ -14,7 +14,9 @@ on:
 jobs:
   test_with_home_assistant:
     name: Integration tests with Home Assistant
+    if: ${{ github.event_name == 'schedule' || matrix.channel != 'dev' }}
     strategy:
+      fail-fast: false
       matrix:
         channel: [stable, beta, dev]
```
Comment (helto4real) [inline .github/workflows/ci_integration_tests.yaml:21]: Should we not at least run the release version on CI ?

Comment (ludeeus) [reply .github/workflows/ci_integration_tests.yaml]: "stable" does that

**Comment by helto4real** [comment]
SUPER!

**Pull request merged**